### ISSUE TITLE
[Bug] Column headers are shifted when responsiveTable is set to true after upgrading to Backpack 5

### ISSUE THREAD

**Issue by Cocker** [Laravel-Backpack/CRUD#4222]
# Bug report

### What I did

Upgraded my existing Backpack 4 project that had `responsiveTable` set to `false` to version 5.

### What I expected to happen

I went through the [Upgrade guide](https://backpackforlaravel.com/docs/5.x/upgrade-guide) and didn't see anything regarding the matter, so I didn't expect this part to break

### What happened

Column headers were slightly shifted and were not aligned with the content (unlike when `responsiveTable` is set to `true`)

`reponsiveTable => true`:
![image](https://user-images.githubusercontent.com/3314103/155324342-a9f6fc73-2d7e-409f-94d4-10e1dccdf22d.png)

`reponsiveTable => false`:
![image](https://user-images.githubusercontent.com/3314103/155324441-05400371-0974-4275-b897-bfb5e744ad78.png)

### What I've already tried to fix it

I tried creating a fresh project and was able to reproduce it just by setting `responsiveTable` to  `false` . It doesn't even require content to overflow the container. I also figured that after upgrading to Backpack 5 two empty rows on top and bottom are now being added when `reponsiveTable => false`. I suspect it to be the cause of the issue.

### Is it a bug in the latest version of Backpack?

After I run ```composer update backpack/crud``` the issue is it still there.

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:

PHP 8.1.2 (cli) (built: Jan 18 2022 23:52:03) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.2, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.2, Copyright (c), by Zend Technologies

### LARAVEL VERSION:
v9.1.0@ca7ddd4782f120ae50569d04eb9e40e52f67a9d9

### BACKPACK VERSION:
5.0.8@f811634e8c4789fa15026d70592515b8051700ad

**Comment by pxpm** [comment]
Hey @tabacitu ! This is the issue I wanted to talk to you about. 

I think I found the problem reported here, but I was not so involved in this css issues and I am a bit lost on how/why some decisions where made.

So what broke this was removing this css rule: https://github.com/Laravel-Backpack/CRUD/commit/73963dac462dcd1b797441457d8d07614858bc37#r67723250

I then followed this changes and it all started in https://github.com/Laravel-Backpack/CRUD/pull/3942 that lead to @promatik  doing that change in https://github.com/DigitallyHappy/BackStrap/commit/268c02080633f3aae7c41e3c7c7fb5be8640279f that was later in part reverted in https://github.com/DigitallyHappy/BackStrap/commit/1930f35df20ca5ad36e5e6b238b639e9155898be

I think it's easier if any of you guys pick this one as I don't know in first place why that rule was removed or if adding it back could have any implication.

Cheers!
